packages/dev/docs/pages/blog/building-a-numberfield-part-1.mdx

```diff
@@ -0,0 +1,64 @@
+<!-- Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+
+import {BlogPostLayout, Video} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+keywords: [react aria, react spectrum, react, spectrum, interactions, numberfield, touch, spinbutton]
+description: Numberfields seem like simple components at first, but they hide a lot of complexity under the hood.
```

Comment:
I love this description, but feel the topics below are ancillary relevant ones. I don't see the below building the story of this description.